airbyte-cdk/python/airbyte_cdk/sources/declarative/extractors/jello.py

```diff
@@ -40,4 +40,7 @@ def __post_init__(self, options: Mapping[str, Any]):
     def extract_records(self, response: requests.Response) -> List[Record]:
         response_body = self.decoder.decode(response)
         script = self.transform.eval(self.config)
-        return jello_lib.pyquery(response_body, script)
+        try:
+            return jello_lib.pyquery(response_body, script)
+        except KeyError:
```

Comment:
```suggestion
        except KeyError:
          # if the requested key is not found then we consider it an empty response instead of failing the sync
```